skylines/frontend/forms/pilot.py

```diff
@@ -118,14 +118,14 @@ class LiveTrackingSettingsForm(Form):
 
 
 class RecoverStep1Form(Form):
-    email_address = EmailField(l_('eMail Address'), validators=[
-        InputRequired(message=l_('Please enter your email address.')),
+    email_address = EmailField(l_('Email Address'), validators=[
+        InputRequired(message=l_('Please enter your Email Address.')),
```

Comment:
this however is not a title and more like a sentence so I would just use "email address" in this case